user_manual/ios_app.rst

```diff
@@ -67,6 +87,52 @@ If the file is filled white, then it is available offline.
 Click the button to make the file available, or not available offline.
```

Comment:
Maybe add an explanation of what means this concept: That will keep your selected file or folder synchronized if any change is made in other client. Your files will always be downloaded and updated.